combineNodes.js

```diff
@@ -1,9 +1,9 @@
 // import state class for instanceof check
-const State = require('./stateNode.js');
+const StateNode = require('./stateNode.js');
 
 // ==================> FOR TESTING ONLY <=================== \\
```

Comment:
We Should Probably Remove this Unused Code.

Reply:
I would suggest moving these into a test file, and we can setup Mocha and Travis.